templates/components/deployments/typescript/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:20-alpine as build
+
+WORKDIR /app
+
+# Install dependencies
+COPY package.json pnpm-lock.yaml* /app/
+RUN npm install 
+
+# Build the application
+COPY . .
+RUN npm run build
+
+# ====================================
+FROM build as release
```

Comment:
looks to me like the release image doesn't have any files (no COPY command) - how does this work?

Reply:
It's already copied in the build stage. We just use everything from the build stage to the release for now.

Reply:
ah i see, should be `FROM node20 as release` and then copy just the build files